src/org/usfirst/frc/team2335/robot/Robot.java

```diff
@@ -197,7 +251,7 @@ public void teleopPeriodic() //This function is called periodically during opera
     		cameraLight.set(Relay.Value.kOff);
     	}
     	
-		Scheduler.getInstance().run();
+    	Scheduler.getInstance().run();
```

Comment:
The Scheduler.getInstange().run(); is an autogenerated thing that is is command based robots, it basically handles for everything runs properly.